aten/src/ATen/native/quantized/cpu/qconv_prepack.cpp

```diff
@@ -284,7 +292,19 @@ class QConvPackWeightInt8 final {
       output_padding.push_back((int64_t)0);
     }
     return _run(weight, bias, stride, padding, output_padding, dilation, groups,
-                false);
+                /*transpose=*/false);
+  }
+
+  static c10::intrusive_ptr<ConvPackedParamsBase<kSpatialDim>> run_deconv(
+      Tensor weight,
+      c10::optional<Tensor> bias,
+      torch::List<int64_t> stride,
+      torch::List<int64_t> padding,
+      torch::List<int64_t> output_padding,
+      torch::List<int64_t> dilation,
+      int64_t groups) {
+    return _run(weight, bias, stride, padding, output_padding, dilation, groups,
+                /*transpose=*/true);
```

Comment:
optional nit, would be good to avoid `/*....*/`, this removes the ability to easily comment out blocks of code for debugging